comp/core/status/render_helpers.go

```diff
@@ -101,8 +105,25 @@ func RenderHTML(templateFS embed.FS, template string, buffer io.Writer, data any
 	if tmplErr != nil {
 		return tmplErr
 	}
+
+	var stdbuff bytes.Buffer
 	t := htemplate.Must(htemplate.New(template).Funcs(HTMLFmap()).Parse(string(tmpl)))
-	return t.Execute(buffer, data)
+	err := t.Execute(&stdbuff, data)
+	_, _ = buffer.Write(stdbuff.Bytes())
+	if err != nil {
+		return err
+	}
+
+	var pkgbuff bytes.Buffer
+	tt := pkghtmltemplate.Must(pkghtmltemplate.New(template).Funcs(HTMLFmap()).Parse(string(tmpl)))
+	terr := tt.Execute(&pkgbuff, data)
+	if terr != nil {
+		log.Warnf("Error executing shadow pkg/template/html for %s: %s", template, terr)
+	} else if pkgbuff.String() != stdbuff.String() {
+		log.Infof("Shadow pkg/template/html output does not match html/template output for %s", template)
```

Comment:
Should we log the difference? Maybe as DEBUG, so customers do not see it, also showing a the does not match message as Info, might trip customers. Since the change is internal to them, maybe we can log both of the messages as warn or debug

Reply:
The issue is that not all differences are unexpected, for example some templates read atomic variables, they can change between two executions, so we can't just warn about it
I think what we care most about is which template has a change, so that we can manually check whether it is using methods, logging a potentially very big template would be spammy...

Reply:
What's the issue with logs ? It's more convenient to print the error 😄 
The main issue we expect here is that templates using methods will fail
Having an info log when there is a diff is just to be safe, but that's not what should happen with this change

Reply:
There is nothing wrong with logging, but how are you planning on monitoring once this gets released? 
Since we can emit internal metrics, we can use those to monitor for these mismatches in a more holistic way than just logging. 

We can create a dashboard, etc.

Reply:
Just search for logs in the app ? (from internal deployments)
The metric wouldn't help with customer cases either, unless we add it to instrumentation telemetry (is that what you had in mind ?)

But even then we can't tag with the whole error message, at best we can tag with the name of the template file
We can discuss this separately but I'd rather having doing that in this PR anyway

Reply:
We agreed that we would use the `agent telemetry` to send metrics regarding discrepancies on the customer side of things. @pgimalac Would add it on a separate PR